src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -4661,6 +4662,221 @@ protected override PSMemberInfoInternalCollection<T> GetMembers<T>(object obj)
     }
 
     /// <summary>
+    /// Deals with DataRow objects
```

Comment:
Can I make this an exception? DataRow/DataRowView adapters were enabled in `MshObject.cs` when I was working on the first cut of refactoring to enable the build. `ExtraAdapter.cs` contains the adapters that are not supported in powershell core and is excluded from the build, that's why I moved those 2 adapters to `CoreAdatper.cs`.